app/utils/md-calc-storage.js

```diff
@@ -0,0 +1,6 @@
+let mdCalcStorage = (obj) => {
```

Comment:
I think this should have a more generic name, e.g. md-object-size, or something like that.

Reply:
Great idea! Changed it to md-object-size and all of its instances